ha/setup/ha_setup.py

```diff
@@ -670,7 +671,8 @@ def process(self):
                 self._remove_node(node_name)
             else:
                 # Destroy
-                self._destroy_cluster(node_name)
+                self._cluster_manager.cluster_controller.destroy_cluster()
+                # self._destroy_cluster(node_name)
```

Comment:
In cleanup 
force should be true

Reply:
Kept default value of force=True in destroy cluster

Reply:
Please delete this commented line.